package.json

```diff
@@ -38,7 +38,8 @@
     "build-kolibri-tools": "yarn workspace kolibri-tools run build-kolibri-tools",
     "publish-packages": "node ./packages/publish.js",
     "hashi-dev": "yarn workspace hashi run dev",
-    "hashi-build": "yarn workspace hashi run build"
+    "hashi-build": "yarn workspace hashi run build",
+    "lint:css": "stylelint '**/*.vue'"
```

Comment:
We should not add a new command. Instead, the linting should be performed by the existing command defined above, `lint-frontend`

Reply:
Thank you for reviewing @indirectlylit. I'll do it in that way